engine/helper/pka_helper.c

```diff
@@ -649,6 +890,24 @@ static void pka_engine_put_handle(pka_engine_info_t *engine)
     reset_pka_handle(*handle);
 }
 
+static void init_mont_curves(void)
```

Comment:
Q Do you think it is worth it to split this function into 2 functions?

static void init_mont_curves(void)
{
  init_mont_curve25519();
  init_mont_curve448();
}

Reply:
Yes, makes sense, I'll split the function into 2.